pkg/apis/apps/v1beta2/zz_generated.conversion.go

```diff
@@ -41,6 +41,10 @@ func init() {
 // Public to allow building arbitrary schemes.
 func RegisterConversions(scheme *runtime.Scheme) error {
 	return scheme.AddGeneratedConversionFuncs(
+		Convert_v1beta2_ControllerRevision_To_apps_ControllerRevision,
```

Comment:
@liyinan926 We'd better separate such auto-generated files in another commit, which will be easy to get rebased.

ping @janetkuo, PTAL #49645 for conversion tests. For this new added `ControllerRevision`, shall we append the tests in this PR.